### PR TITLE
docs: notification

### PR DESCRIPTION
**Pull request by adityathebe**
Resolves: https://github.com/flanksource/docs/issues/93

### CODE REVIEW

File: docs/notifications/concepts/notification.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Notification allows you to receive alerts on a preferred channel when a particular set of events occur.
+
+![Create notification form](../../images/create-notification-form.png)
```
Comment (moshloop) [inline docs/notifications/concepts/notification.md:3]: I don't think it is necessary to add screenshots as then we will need to update them

File: docs/notifications/events/components.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+| `component` | The component object          | [`Component`](#component) |          |
+| `agent`     | Agent details (if applicable) | [`Agent`](#check)         | `true`   |
+| `permalink` | A link to the health check    | `string`                  |          |
+
+### Component
+
+| Field                | Description                                          | Schema              | Optional |
+| -------------------- | ---------------------------------------------------- | ------------------- | -------- |
+| `id`                 | The id of the component                              | `uuid`              |          |
+| `topology_id`        | The id of the topology that generated this component | `uuid`              |          |
```
Comment (moshloop) [inline docs/notifications/events/components.md:22]: I think we can remove agent_id - also the selectors, is_leaf, path etc - Basically anything that wouldn't make logical sense to filter on  or include in a message

File: docs/notifications/events/components.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+| `description` | Short description of the agent        | `string`            |          |
+| `ip`          | The ip of the agent                   | `string`            |          |
+| `version`     | The version of the agent              | `string`            |          |
+| `username`    | The username of the agent             | `string`            |          |
+| `person_id`   | The person associated with this agent | `uuid`              | `true`   |
+| `properties`  | The properties of the agent           | `map[string]string` | `true`   |
+| `tls`         | The tls of the agent                  | `string`            |          |
+| `created_by`  | The created by of the agent           | `uuid`              |          |
+| `created_at`  | The created at of the agent           | `time.Time`         |          |
+| `updated_at`  | The updated at of the agent           | `time.Time`         |          |
```
Comment (moshloop) [inline docs/notifications/events/components.md:75]: Do we have all of these fields? It probably also doesn't make sense to haver more than id and name

Comment (adityathebe) [reply docs/notifications/events/components.md:75]: okay. getting rid of them. We can always add them if the need arises.

**Pull request merged**